docs/src/appendix/cast/multicall.md

```diff
@@ -27,4 +34,19 @@ call_type = "invoke"
 contract_address = "map_contract"
 function = "put"
 inputs = ["0x123", "234"]
-```
\ No newline at end of file
+```
+
+## `new`
+
+Generates an empty template for the multicall `.toml` file that may be later use with the `run` subcommand. It either outputs it to a new file or to the standard output.
+
+### `--output_path, -p <PATH>`
+Optional.
+
+When provided, it specifies where the template should be saved, it has to be a file path.
+If omitted, the template contents are going to be printed out to the stdout.
+
+### `--overwrite, -o <OVERWRITE>`
+Optional.
+
+Overwrites the file specified by `--output-path`, if it exists.
```

Comment:
I thought that we agreed on different wording, to ensure noone thinks 'it' refers to the flag (but the file)?
https://github.com/foundry-rs/starknet-foundry/pull/98#discussion_r1268035087

Reply:
[fix dox](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/cc78ee39dbe5a143333953db5261d46c6cce69b8)